app/_locales/en/messages.json

```
@@ -1176,6 +1176,12 @@
   "searchTokens": {
     "message": "Search Tokens"
   },
+  "searchAccounts": {
+    "message": "Search Accounts"
+  },
+  "noAccountsFound": {
+    "message": "No accounts found for a given search query"
```

```suggestion
    "message": "No accounts found for the given search query"
```
Maybe this wording would be better?

---

agree `the` is more appropriate here